### PR TITLE
Implements multiple assignees for new issues as well

### PR DESCRIPTION
**Pull request by discorick**
Allows multiple assignees to be added to a new issue

### CODE REVIEW

File: ember-app/app/controllers/issue/create.js [hunk 1]
```diff
@@ -18,6 +18,10 @@ var IssuesCreateController = Ember.Controller.extend({
   }.observes('selectedRepo'),
   otherLabels: Ember.computed.alias("selectedRepo.data.other_labels"),
   assignees: Ember.computed.alias("selectedRepo.data.assignees"),
+  //Ugly Hack for backwards Enterprise Compatibiltiy
+  assigneesEnabled: function(){
+    return this.get("selectedRepo.board.issues").isAny("assignees");
```
Comment (dahlbyk) [inline ember-app/app/controllers/issue/create.js]: This seems likely to be slow on large repos without `assignees` support. Can we just look at the first one?

**Comment by dahlbyk** [comment]
Looks good other than the concern about perf when assignees isn't supported. I'd still love to be able to configure the review app to enable/disable the assignees preview to test each scenario.

File: ember-app/app/controllers/issue/create.js (hunk 1 above)
Comment (discorick) [inline ember-app/app/controllers/issue/create.js]: The base implementation for `any` short-circuits on `found` https://github.com/emberjs/ember.js/blob/v2.6.0/packages/ember-runtime/lib/mixins/enumerable.js#L662

Comment (dahlbyk) [inline ember-app/app/controllers/issue/create.js]: Right. It's efficient when `assignees` is expected to be found, but will scan every issue looking for `assignees` when it's not found.

Comment (discorick) [inline ember-app/app/controllers/issue/create.js]: No bueno on this, GitHub API returns issues with any assignees without an assignees key :(

**Comment by discorick** [comment]
Alight added a guard for old client code here: 99cf61a

**Pull request merged**